tests/k8s/oracle.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: oracle
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: oracle
+    spec:
+      containers:
+      - name: oracle
+        image: beckmani/oracle_server:latest
```

Comment:
please move docker container to a dedicated spacemesh docker repo

Reply:
Done. See:
https://cloud.docker.com/u/spacemeshos/repository/docker/spacemeshos/oracle